charts/ibm-object-storage-plugin/Chart.yaml

```diff
@@ -28,5 +28,5 @@ keywords:
 maintainers:
   - name: IBM
     email: contsto2@in.ibm.com
-#kubeVersion: '>=1.10.1'
+# kubeVersion: '>=1.10.1'
```

Comment:
nit: no-op change

Reply:
This change was needed to allow chart-testing to pass